### PR TITLE
Login

### PR DESCRIPTION
**Pull request by danhmanh**
image

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```
Comment (itssomething) [inline app/helpers/sessions_helper.rb]: ()

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -25,10 +25,16 @@
...
-        <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
-        <%= t ".manage_account" %>
+
+
+      <% if current_user? @user %>
+        <%= link_to edit_user_path, class: "btn btn-primary center-block" do %>
+          <i class="fa fa-pencil-square-o"></i> <%= t ".manage_account" %>
+        <% end %>
+      <% else %>
+        <a href="#" class="btn btn-warning center-block">Follow</a>
```
Comment (itssomething) [inline app/views/users/show.html.erb]: i18n

**Comment by itssomething** [comment]
ok

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -53,8 +53,11 @@ li { list-style-type:none;}
   }
 }
 
-.navbar-left.img-responsive{
-  img{
-    max-width: 150px;
-  }
+.logo {
+    height: 50px;
```
Comment (thanhhungchu95) [inline app/assets/stylesheets/custom.scss]: 2 space thôi

File: app/controllers/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      flash[:success] = t ".login_success"
+      redirect_to user
+    else
+      flash.now[:danger] = t ".login_fail"
+      render "new"
```
Comment (thanhhungchu95) [inline app/controllers/sessions_controller.rb]: render :new

**Pull request merged**